unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraStateFPS.cs

```diff
@@ -45,6 +45,8 @@ private void OnMovingPlatformStart(bool current, bool previous)
         }
         private void OnMovingPlatformRotate(Quaternion current, Quaternion previous)
         {
+            // Disabled camera rotation on moving platforms because this was causing issues for some scenes
```

Comment:
If this method is causing issues we should just remove it entirely.

Reply:
Please track an issue to re-enable the feature, removing a behavior may be a good temporary fix for the helicopter situation but we should restore the original behavior

Reply:
I would either remove it (but we will need to re-implement it again once we have a new `playerRoation` or similar exposed for the SDK) or comment it out with a clear comment that it has to be uncommented when we have that new property to use.